step.sh

```diff
@@ -109,6 +112,34 @@ function get_upload_status {
     echo "$upload_status"
 }
 
+function get_run_result {
+    local run_arn="$1"
+    validate_required_variable "run_arn" $run_arn
```

Comment:
`$run_arn` should probably be quoted for safety (`"$run_arn"`) ... looks like I forgot to do this in my own validation code as well.